docs/design_principles.md

```diff
@@ -0,0 +1,76 @@
+<style>
```

Comment:
Since it is a markdown I would look how to add headers in [markdown/basic syntax](https://www.markdownguide.org/basic-syntax/). Markdown will interpret the # to an h1 header or the ### to an h3 header.

I would remove the <style> tag that adds the spacing. Add an h1 header as: `# Design Principles`